api/src/main/java/org/eclipse/microprofile/config/inject/ConfigProperty.java

```diff
@@ -125,4 +127,25 @@
      */
     @Nonbinding
     String defaultValue() default UNCONFIGURED_VALUE;
+
+    /**
+     * @see org.eclipse.microprofile.config.ConfigAccessor#evaluateVariables(boolean)
+     * @return whether variable replacement is enabled. Defaults to {@code true}.
+     */
+    @Nonbinding
+    boolean evaluateVariables() default true;
+
+    /**
+     * Only valid for injection of dynamically readable values, e.g. {@code Provider<String>}!
+     * @return {@code TimeUnit} for {@link #cacheFor()}
+     */
+    @Nonbinding
+    TimeUnit cacheTimeUnit() default TimeUnit.SECONDS;
```

Comment:
I thought we agreed to use ChronoUnit. MP FT uses ChronoUnit. It is better to sync up.